### DISCUSSION TITLE
2022/01/15/fursona-story-extra-2/

### DISCUSSION THREAD

**Discussion by giscus** [su226/su226.github.io#7] [Announcements]
# 2022/01/15/fursona-story-extra-2/

还是一个无所事事的下午，我坐在电脑前一边吃薯片，一边四处点来点去。突然，开门的声音响起，紧接着似乎有什么东西重重地摔在了地上。当我赶到门口时，伊哈根正扶着墙，颤抖不止。 看到我来了，他还是露出了微笑，硬撑着说出了“主人，我……我没事，真的。”我并不知道发生了什么，但扒开他略显肮脏的皮毛，有几道还隐约在渗血的伤口，还有数不清的红印。 虽然他坚持不告诉我到底发生了什么，但我的直觉还是立马告诉我，他又去

https://su226.tk/2022/01/15/fursona-story-extra-2/

**Comment by GreenZeFurry** [comment]
啊哈哈哈哈兽兽来咯
